pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1782,3 +1804,37 @@ func getServingCertsMode(remoteKubernetes *ispnutil.Kubernetes) string {
 	}
 	return ""
 }
+
+func (r *ReconcileInfinispan) applyLabelsToCoordinatorsPod(podList *corev1.PodList, infinispan *infinispanv1.Infinispan, cluster ispnutil.ClusterInterface, logger logr.Logger) bool {
+	pass, err := cluster.GetPassword(consts.DefaultOperatorUser, infinispan.GetSecretName(), infinispan.GetNamespace())
```

Comment:
@rigazilla , looks like we will never know when err will return from cluster.GetPassword

Reply:
yep, let me add a log